yarn.lock

```diff
@@ -9689,7 +9674,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"node-fetch@npm:2.6.7, node-fetch@npm:^2.6.1, node-fetch@npm:^2.6.6, node-fetch@npm:^2.6.7":
+"node-fetch@npm:2.6.7, node-fetch@npm:^2.6.1, node-fetch@npm:^2.6.7":
```

Comment:
Do we need `node-fetch@npm:^2.6.1` to go away as well?

Reply:
I _think_ this line is specifying the versions as they're requested by various project dependencies (direct or indirect) that will be resolved to this specific version of the dependency. In theory, I think this would allow a block like:
```
"node-fetch@npm:^3.0.0":
  version: 3.0.1
  resolution: "node-fetch@npm:3.0.1"
  ...
```
to coexist in the same project, and each library could get the major version of node-fetch that it wants.